src/components/charts/chart-builder.js

```diff
@@ -1,17 +1,25 @@
-const colorSchemes = {
-  'default': ['rgba(46, 139, 87, 0.8)', 'rgba(220, 20, 60, 0.8)', 'rgba(64, 224, 208, 0.8)', 'rgba(192, 192, 192, 0.8)', 'rgba(255, 218, 185, 0.8)', 'rgba(75, 0, 130, 0.8)'],
-  'redGreen': ['rgba(255, 215, 0, 0.8)', 'rgba(220, 20, 60, 0.8)', 'rgba(173, 200, 255, 0.8)', 'rgba(192, 192, 192, 0.8)', 'rgba(244, 164, 96, 0.8)', 'rgba(0, 0, 128, 0.8)'],
-  'greyScale': ['rgba(0, 0, 0, 0.8)', 'rgba(47, 79, 79, 0.8)', 'rgba(119, 136, 153, 0.8)', 'rgba(128, 128, 128, 0.8)', 'rgba(192, 192, 192, 0.8)', 'rgba(220, 220, 220, 0.8)']
+export const colorSchemes = {
+  'default': {
+    name: 'Normal',
+    colors: ['rgba(46, 139, 87, 0.8)', 'rgba(220, 20, 60, 0.8)', 'rgba(64, 224, 208, 0.8)', 'rgba(192, 192, 192, 0.8)', 'rgba(255, 218, 185, 0.8)', 'rgba(75, 0, 130, 0.8)']
+  },
+  'redGreen': {
+    name: 'Sagi Mode',
```

Comment:
Please find out the name of Sagi's illness and name this color appropriately.

Reply:
I have been specifically requested to call it Sagi Mode

Reply:
I doubt that the one who requested that was serios. please rename it to "Deutranopia" mode. that is the name of Sagi's illness.

Reply:
Done